index.html

```diff
@@ -4956,104 +4956,129 @@ <h2>Herd Privacy</h2>
-irreversibly associate services that are not associated in any other context. This
-correlation on its own may lead to de-anonymization, revealing information about the
-<a>DID subject</a> even if the <a>URIs</a> used did not.
-      </p>
-      <p>
-Third, because some types of <a>DID subjects</a> may be more or less likely to list specific
-endpoints, e.g., a <a>DID</a> for an automobile may include a pointer to a public title
-record at the Department of Motor Vehicles, while a <a>DID</a> for an individual would not.
-As such, the listing of a given service could, by itself, leak information that can
-be used to infer something about the subject.
-      </p>
-      <p>
-It is the goal of herd privacy to ensure that the nature of specific subjects is
-obscured by the population of the whole. To maximize herd privacy, implementers
-need to rely on one &mdash; and only one &mdash; service endpoint, with that endpoint providing
-a proxy or mediator service that the controller is willing to depend on to protect
-such associations and that blinds requests to the ultimate service.
-      </p>
-      <p>
-Toward this end, consider any of the following:
-      </p>
-      <ul>
-        <li>
+        <p>
+First, because service endpoints might be specified as <a>URIs</a>, they could
+unintentionally leak personal information because of the architecture of the
+service. For example, a service endpoint of
+<code>http://example.com/MyFirstName</code> is leaking the term
+<code>MyFirstName</code> to everyone who can access the <a>DID document</a>.
+When linking to legacy systems, this is an unavoidable risk and care is expected
+to be taken in such cases. This specification encourages new, <a>DID</a>-aware
+endpoints to use nothing more than the <a>DID</a> itself for any identification
+necessary. For example, if a service description were to include
+<code>http://example.com/did%3Aexample%3Aabc123</code>, no harm would be done
+because <code>did:example:abc123</code> is already exposed in the DID Document;
+it leaks no additional information.
+        </p>
+
+        <p>
+Second, because a <a>DID document</a> can list multiple service endpoints, it is
+possible to irreversibly associate services that are not associated in any other
+context. This correlation on its own may lead to privacy harms by revealing
+information about the <a>DID subject</a> even if the <a>URIs</a> used did not
+contain any sensitive information.
+        </p>
+        <p>
+Third, because some types of <a>DID subjects</a> might be more or less likely to
+list specific endpoints. For example, a <a>DID</a> for an automobile might
+include a pointer to a public title record at the Department of Motor Vehicles,
+while a <a>DID</a> for an individual would not include that information. As
+such, the listing of a given service could, by itself, leak information that can
+be used to infer something about the <a>DID subject</a>.
```

Comment:
```suggestion
Third, because some types of <a>DID subjects</a> might be more or less likely to
list specific endpoints, the listing of a given service could, by itself, leak
information that can be used to infer something about the <a>DID subject</a>.
For example, a <a>DID</a> for an automobile might include a pointer to a public
title record at the Department of Motor Vehicles, while a <a>DID</a> for an
individual would not include that information.
```